### DISCUSSION TITLE
Can not connect

### DISCUSSION THREAD

**Discussion by dewaynemeats** [arkypita/LaserGRBL#1031] [Q&A]
I am using com 4
I click on connect and will not connect. says not connected
Help
Dewayne

**Comment by Maxoch** [comment]
Проверте целостность кабеля, проверте установлен ли драйвер. К другому ПО пробовали подключаться?

**Comment by gmmanonymus111** [comment]
Are you sure your machine is connected to COM4? Does Baudrate set to the correct value?
Does your machine uses GRBL?
Can the USB cable transfer data, or is it just a charging cable?

**Comment by dewaynemeats** [comment]
I am using comm4The cable that came with the machineI do not know what Baudrate is.

Sent from Yahoo Mail on Android 
 
  On Wed, Jun 3, 2020 at 8:33, Alex<notifications@github.com> wrote:   


Are you sure your machine is connected to COM4? Does Baudrate set to the correct value?
Does your machine uses GRBL?
Can the USB cable transfer data, or is it just a charging cable?

**Comment by gmmanonymus111** [comment]
Baud rate is the 'communication speed' between your PC and Arduino
You can set it here
![image](https://user-images.githubusercontent.com/9658243/83668079-b4e5e500-a5cf-11ea-8d65-5d69b826ddd7.png)
It is usually `9600` or `115200`
Could you tell us more about your machine? Type, GRBL version (if known) etc?
